backend/src/main/java/com/rota/database/orm/engagement/Engagement.java

```diff
@@ -0,0 +1,34 @@
+package com.rota.database.orm.engagement;
+
+import com.rota.database.orm.staff.Staff;
+import java.time.Instant;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@Entity
+@AllArgsConstructor
+@NoArgsConstructor
+public class Engagement {
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  int id;
```

Comment:
To make this easier for someone unfamiliar with the codebase to read I would add an empty line between each field (but not between an annotation and the field it is related to)